python/py-wmctrl/Portfile

```diff
@@ -0,0 +1,26 @@
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-wmctrl
+version             0.3
+platforms           darwin
+license             BSD
+maintainers         nomaintainer
+
+description         A tool to programmatically control windows inside X
+long_description    ${description}
+
+homepage            https://bitbucket.org/antocuni/wmctrl/src/default/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  3bb58717433e5062c44b95349af4dabf21afca37 \
+                    sha256  d806f65ac1554366b6e31d29d7be2e8893996c0acbb2824bbf2b1f49cf628a13 \
+                    size    2505
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+}
```

Comment:
please add ```    livecheck.type  none``` within the ```if```-block; we typically only do a livecheck for the ```py-<portname>```

Reply:
Done